### DISCUSSION TITLE
To run a 2nd  instance of the plugin for two scheduled possibilities?

### DISCUSSION THREAD

**Discussion by jsmdy** [Cloud9Developer/Jellyfin-Newsletter-Plugin#123] [Ideas]
I was thinking that it just need to duplicate and rename the plugin into the repository, but this maybe utopic.
But this way it be can send to different email at different scheduled sending

**Comment by tanichols** [comment]
You can add multiple schedules to the scheduled tasks. That would use the same settings such as to list, and format, but maybe that works for you.
